app/controllers/admin/publishers_controller.rb

```diff
@@ -41,6 +42,12 @@ def update
     redirect_to admin_publisher_path(@publisher)
   end
 
+  def destroy
+    PublisherRemovalJob.perform_later(publisher_id: @publisher.id)
```

Comment:
Question, should we consider making this a function? it seems like these will be one-off calls and don't need to be scheduled. Will defer to your judgement here @yachtcaptain23

Reply:
Sort of intended since I'd imagine there would be more API calls to delete more PII. By locking it all in the same thread, we can see which job succeeded and failed in Sidekiq.